front-end-web/src/Components/Pages/ExportPage.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { Button, TextField } from "@material-ui/core";
 
 /**
  * TODO: Add Documentation
  */
+
 const ExportPage: React.FunctionComponent<Props> = ({
   state,
   handlers,
   classes
 }) => {
+  document.cookie = "stage=4";
   return (
     <Fragment>
+      <Button
+        variant="contained"
+        onClick={function() {
```

Comment:
This should call the handler handlers.handleCreateExportCalendar(name) (which we still need to write). Can you add that as if we did have it?